static/docs/tutorials/interactive.md

```diff
@@ -0,0 +1,15 @@
+# Interactive Tutorials
+
+Learn how DVC can be used in simple ML scenarios:
+
+1. [Data Versioning](https://katacoda.com/dvc/courses/tutorials/versioning)
+   <br/> Using DVC commands to work with multiple versions of datasets and ML
+   models.
+
+2. [Stages and Pipelines](https://katacoda.com/dvc/courses/tutorials/pipelines)
+   <br/> Using DVC commands to build a simple ML pipeline.
+
+3. [dvc fetch](https://katacoda.com/dvc/courses/examples) <br/> We will use an
```

Comment:
`dvc fetch` looks strange a bit to a link by its own. Also , the path `courses/examples` has nothing to do with fetch.  If there is a plan to make a scenario per DVC command we should come up with a better name. Even Command Reference might work better.